content/operate/rc/changelog/june-2024.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: Redis Cloud changelog (June 2024)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: New features, enhancements, and other changes added to Redis Cloud during
+  June 2024.
+highlights: Redis Cloud Essentials support for Terraform
```

Comment:
@cmilesb, please write also that we support modules for A-A databases.
Plus, let's write something like "full version content can be found [here](https://github.com/RedisLabs/terraform-provider-rediscloud/blob/main/CHANGELOG.md)."

Reply:
Just fixed in e1eb0ad.